velox/functions/lib/Repeat.cpp

```diff
@@ -109,7 +123,7 @@ class RepeatFunction : public exec::VectorFunction {
         BaseVector::wrapInDictionary(nullptr, indices, totalCount, args[0]));
   }
 
-  VectorPtr applyFlat(
+  VectorPtr applyNonConstant(
```

Comment:
nit: perhaps, rename both methods to applyConstantCount and applyNonConstantCount for clarity. 'constness' applies only to 'count' argument, not to all arguments.